### PR TITLE
Cleanup code and tests for package lifecycle work

### PR DESCRIPTION
**Pull request by kddejong**
*Issue #, if available:*

*Description of changes:*
- Cleanup code and tests for package lifecycle work

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by mrinaudo-aws — APPROVED** [review]
LGTM

File: python/rpdk/typescript/templates/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     },
```
Comment (HeatherFlux) [inline python/rpdk/typescript/templates/package.json:18]: Class Transformer is still 0.3.1 here and needs to be bumped to 0.5.1

File: python/rpdk/typescript/templates/models.ts [hunk 2]
```diff
@@ -33,7 +31,7 @@ export class {{ model|uppercase_first_letter }} extends BaseModel {
     @Type(() => {{ inner_type.type }})
     {% else %}
     @Transform(
-        (value: any, obj: any) =>
```
Comment (HeatherFlux) [inline python/rpdk/typescript/templates/models.ts:36]: `@transform` should look like this 
```
    @Transform(
        (params: TransformFnParams) =>
            transformValue({{ inner_type.wrapper_type }}, '{{ name|lowercase_first_letter|safe_reserved }}', params.value, params.obj, [{{ inner_type.classes|join(', ') }}]),
        {
            toClassOnly: true,
        }
    )
```

File: python/rpdk/typescript/templates/models.ts [hunk 3]
```diff
@@ -4,8 +4,6 @@ import { Exclude, Expose, Type, Transform } from 'class-transformer';
```
Comment (HeatherFlux) [inline python/rpdk/typescript/templates/models.ts:4]: Import from class-transformer should be
```
import { Exclude, Expose, Type, Transform, TransformFnParams } from 'class-transformer';
```

File: python/rpdk/typescript/templates/package.json [hunk 4]
```diff
@@ -8,7 +8,7 @@
         "dist"
     ],
     "scripts": {
-        "build": "npx tsc",
+        "build": "npx tsc --skipLibCheck",
         "prepack": "npm run build",
         "test": "echo \"Error: no test specified\" && exit 1"
     },
```
Comment (HeatherFlux) [inline python/rpdk/typescript/templates/package.json:14]: lib_path is still set to 1.0.1 and needs to be updated to 1.0.6 in the codegen.py file

File: python/rpdk/typescript/codegen.py [hunk 5]
```diff
@@ -305,6 +305,7 @@ def _build(self, base_path):
                 )
```
Comment (HeatherFlux) [inline python/rpdk/typescript/codegen.py:305]: Line 29 of this file is still at version 1.0.1 and needs to be at 1.0.6

**Review by HeatherFlux — COMMENTED** [review]
I found a few bugs while testing. What is worse is this no longer works as a package due to the way that the models.ts file in templates was updated.

I tried to create a PR but I don't have access to create PRs

**Comment by kddejong** [comment]
> I found a few bugs while testing. What is worse is this no longer works as a package due to the way that the models.ts file in templates was updated.
> 
> I tried to create a PR but I don't have access to create PRs

Let's get you back up and running.  You can't fork and submit a PR?

**Pull request merged**